compile-reproducer.sh

```diff
@@ -0,0 +1,65 @@
+#! /bin/bash
+
+usage() {
+        echo "usage: $(basename "$0") [-h|--help] <clients_version> [clients_build]"
+}
+
+if [ $# -lt 1 ] || [[ "$@" = *"-h"* ]]; then
+        usage
+        exit 0
+fi
+
+TARGET_DIR=./dist/tcpkeepalivesettings-reproducer
+REPRODUCER_DIR=./TcpKeepAliveSettings-reproducer/TcpKeepAliveSettings
+CLI_VER=$1
+CLI_BUILD=${2:-""}
+
+if [[ "$CLI_BUILD" != "" ]]; then
+	CLI_BUILD=".$CLI_BUILD"
+fi
+
+# Create directory for finished product
+rm -rf $TARGET_DIR
+mkdir -p $TARGET_DIR
+
+# Search and replace Package AMQPNetLite.Core with AMQP.dll
+sed -i 's/PackageReference Include="AMQPNetLite.Core" Version="2.3.0/Reference Include="AMQP.dll/g' $REPRODUCER_DIR/TcpKeepAliveSettings.csproj
```

Comment:
OK, I see that you do change here which relates to my comment above.

Reply:
This actually leads me to the rhetorical question: Why am I using sed every time, why don't I just change it...

- [ ] Change References inside the project file, remove sed command from the build script